torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1126,30 +1125,8 @@ distributed: (:class:`~torch.distributed.ProcessGroupGloo.Options` and
   shared_ptr_class_<::gloo::transport::Device>(processGroupGloo, "Device");
 
   intrusive_ptr_class_<::c10d::ProcessGroupGloo::Options>(
-      processGroupGloo,
-      "Options",
-      processGroupOptions,
-      R"(
-ProcessGroup options for Gloo backend
-
-Arguments:
-    timeout (timedelta, optional): Timeout for operations executed against
-            the process group. Default value equals 30 minutes.
-
-Example::
-    >>> import torch.distributed as dist
-    >>> import tempfile
-    >>> from datetime import timedelta
-    >>>
-    >>> temp_name = tempfile.NamedTemporaryFile(delete=False).name
-    >>> gloo_options = dist.ProcessGroupGloo.Options(timeout=timedelta(seconds=300))
-    >>> # initialize a gloo process group with the options just created
-    >>> dist.init_process_group("gloo", init_method=f"file://{temp_name}",
-    >>>                          world_size=2, rank=0, pg_options=gloo_options)
-      )")
-      .def(
-          py::init<std::chrono::milliseconds>(),
-          py::arg("timeout") = kProcessGroupDefaultTimeout)
+      processGroupGloo, "Options", processGroupOptions)
```

Comment:
Since `c10d.ProcessGroupGloo.Options` is no longer supported should we prefix it with a _? `c10d.ProcessGroupGloo._Options`?